src/app/pages/auth/login/LoginView.js

```diff
@@ -14,13 +16,28 @@ LoginView.defaultProps = {
   valid: true,
 }
 
+// ussage with variables
+const title = gettext('Login')
+
 export default function LoginView({ handleSubmit, submitting, valid }) {
+  useEffect(() => {
+    notification.success({
+      description: gettext('message description'),
+    })
+  }, [])
+
   return (
     <form onSubmit={handleSubmit} className={classNames.login}>
-      <h2>Login</h2>
-      <TextField name="email" label="Email" type="email" />
-      <TextField name="password" label="Password" type="password" />
-      <button disabled={submitting || !valid }>Login</button>
+      <h2>{title}</h2>
+      <TextField name="email" label={gettext('Email')} type="email" />
+      <TextField name="password" label={gettext('Password')} type="password" />
+      <button disabled={submitting || !valid }>{gettext('Login')}</button>
+      <span dangerouslySetInnerHTML={{
+        __html: interpolate(
+          pgettext('EMAIL VALIDATION', 'It looks like an account with this email address already exists. Please %slog in%s to continue'),
```

Comment:
`pgettext` used to get string for context and should be normally used inside `interpolate`

Reply:
HOOK, HOC Render prop